### DISCUSSION TITLE
How Could I fix this issue

### DISCUSSION THREAD

**Discussion by alshoikan** [NomicFoundation/hardhat#3657] [Troubleshooting]
I face this message:

[Error: EACCES: permission denied, open '/cache/solidity-files-cache.json'] {
  errno: -13,
  code: 'EACCES',
  syscall: 'open',
  path: '/cache/solidity-files-cache.json'
}

When I run npx hardhat compile 

How could I fix it ?

**Comment by fvictorio** [comment]
It seems like Hardaht is trying to use `/cache` (a directory in the root directory) as the cache path, and you need root access to read/write it. Maybe you have a config that specifies `/cache` as the path instead of `cache` or `./cache`?
